src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -37,69 +39,72 @@ If you're more adventurous or want to get the latest changes of the xk6-browser
 
 <InstallationInstructions extensionUrl="github.com/grafana/xk6-browser"/>
 
+## Launch
+
+The first step is to launch a [Browser](/javascript-api/xk6-browser/browser). After the browser starts, you can interact with it using the [browser-level APIs](#browser-level-apis). The only accepted browser type is currently `chromium`.
+
+| Method                         | Description                                                                                                                                                                                                         |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| launch(browserName, [options]) | The first parameter is required, and the only accepted `browserName` is currently `chromium`. The `options` parameter is optional. You can see a list of all possible `options` in [the options heading](#options). |
+
+### Options
+
+| Method            | Description                                                                                                                                                                                              |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| args              | Extra command line arguments to include when launching browser process. See [this link](https://peter.sh/experiments/chromium-command-line-switches/) for a list of Chromium arguments. Note that arguments should not start with `--` (see the [example](#Example)).  |
+| debug             | All CDP messages and internal fine grained logs will be logged if set to `true`. Default is `false`.                                                                                                     |
+| devtools          | Open up developer tools in the browser by default. Default is `false`.                                                                                                                                   |
```

Comment:
WDYT about adding a "Default" column? [We do this on some pages](https://k6.io/docs/using-k6/scenarios/executors/ramping-vus/) (though I would place the column after "Type" instead), and it makes it easy to see at a glance.

Reply:
That makes sense. I might do that in another PR though so that I can add default columns to all tables i've added/amended.

Reply:
Issue created: https://github.com/grafana/xk6-browser/issues/473

Reply:
Resolved in: c7fafbff787583dc59e4799d1bd4c5c21ef69a78